bundles/config/org.eclipse.smarthome.config.discovery/src/main/java/org/eclipse/smarthome/config/discovery/inbox/Inbox.java

```diff
@@ -119,5 +121,16 @@
      * @param listener the listener to be removed (could be null)
      */
     void removeInboxListener(InboxListener listener);
+    
+    /**
+     * Creates new {@link Thing} and adds it to the {@link ThingRegistry}. 
+     * 
+     * @param thingUID 
```

Comment:
IMHO we could stop using line breaks after the parameter name.
As we configured the formatter we disabled the removal of manual line breaks in JavaDoc becuase it would result in a lot changes. For new could I would prefer to comment without such line break (don't know if we should use a defined style now).